compiler/tests/sct-checker/common.ml

```diff
@@ -9,6 +9,7 @@ module Arch =
        (module Arch_full.Arch_from_Core_arch (C) : Arch_full.Arch))
 
 let load_file name =
+  Utils.nowarning ();
```

Comment:
This looks like a bad idea to me. When the warning about unused local variables was introduced, it helped improving the test suite.

Reply:
Fixed